scripts/packergen.py

```diff
@@ -81,8 +81,14 @@
 systemctl enable rsyslog.service
 
 echo "--> Stopping Google services..."
-systemctl stop google-accounts-daemon.service
-systemctl stop google-instance-setup.service
```

Comment:
These services are legacy for at least a few months now.
https://github.com/GoogleCloudPlatform/compute-image-packages/pull/858

That means that the images we have been building do have the functionality that was supposedly disabled here. 
Why do we disable the instance-setup piece?

Reply:
> > > > The risk now is disabling a functionality that is needed by the packaged software. We need to understand the impact.
> > > 
> > > 
> > > We can select the functionalities to disable:
> > > https://github.com/GoogleCloudPlatform/compute-image-packages/tree/master/packages/python-google-compute-engine#configuration
> > 
> > 
> > Does this file needs to be created on instances? Will we left him on image? Can we create it, disable the services and delete it?
> 
> I think that should do the trick. I am giving it a try now.

If we want to disable only during the execution of packer steps, we need to delete the .cfg file. Btw, the .template commands are outdated, we are supposed to update the file directly.

Reply:
If we don't need these functionalities during the packer steps, it is safer to just disable the service. In this case, we are good as long as the building succeeds.

Tinkering with the config has a higher risk because unintentional changes could be propagated to the final image and interfere with the agent when vm boots.